docs/user_guide.md

```diff
@@ -24,110 +26,144 @@ $ java -jar RepoSense.jar -config CSV_path.csv -output output_path/ -since 01/10
+You can use `Start Tags` to mark the start of your contribution. The author specified in the `Start Tags` will be recognized by RepoSense as the author for all lines between a `Start Tag` and the next `End Tag`. If RepoSense cannot find a matching `End Tag` for a `Start Tag` in the same file, it will assume that all lines between the `Start Tag` to the end of the file is authored by the author specified in the `Start Tag`.
 
 
 ## Dashboard
+The `Dashboard` is written in HTML and Javascript as static pages - readable by majority of web browsers, and easily deploy-able in most hosting platforms (such as [GitHub Pages](https://pages.github.com/)).
 
-The dashboard is written in HTML and Javascript, so you can easily publish and share it. Below is what the Dashboard looks like:
+Below is an example of how the Dashboard looks like:
 
 ![dashboard](images/dashboard.png)
 
-It is consisted of three main parts: tool bar, Chart Panel and Code Panel.
-### Tool Bar
+It consists of three main parts: [Tool Bar](#tool-bar), [Chart Panel](#chart-panel) and [Code Panel](#code-panel).
 
-The tool bar at the top provides a set of filters that control the chart panel. From right to left, the filters are:
-- Sorting: Users can sort by:
-	- Total Contribution: the amount of lines written by the author did in the latest version of the project
-	- Variance: The variance of contribution of all commits. This indicates whether the author is contributing regularly or just rushing before deadlines and milestones. This can be useful for instructors of student projects, e.g. CS2103
-	- Author Name
-	- Team Name: The name of the organization of the repository
-- Interval: Interval refers to amount of time one single ramp represents. Users can choose two modes of time intervals: weekly (7 days)  and daily (1 day).
-- Period: The time period that the ramp charts display.
-- Group By Repo: Checkbox. If checked, the author that contributed to the same repo will be displayed next to each other, no matter what the sorting element is.
-- Search: Only display the author whose name or the repository’s name match the keyword. The user can separate keywords by spaces, and the keywords are logically connected with OR operators.
-- Bookmarking: By clicking the hyperlink icon on the top right corner, a link to the report with all the tooltip settings will be generated and copied to user’s clipboard.
+### Tool Bar
+The `Tool Bar` at the top provides a set of filters that control the chart panel. From right to left, the filters are:
+- Sort : Users can sort by:
+    - Total Contribution : The amount of lines written by the author did in the latest version of the project.
```

Comment:
The amount of lines written by the author ~~did~~ in the latest version of the project.